plugins/eos/gs-plugin-eos.c

```diff
@@ -1728,6 +1746,9 @@ gs_plugin_refine (GsPlugin		*plugin,
 
 		if (gs_plugin_eos_blacklist_app_for_remote_if_needed (plugin, app))
 			continue;
+		
+		if (gs_plugin_eos_remove_blacklist_from_usb_if_needed (plugin, app))
+			continue;
```

Comment:
We shouldn't continue here, because the app may need to be refined as popular, right? The other blocks call continue because they've just blacklisted an app, so there's no point in wasting time refining it.